make/mt-os/mt-cc3220.mk

```diff
@@ -105,17 +105,19 @@ XI_COMPILER_FLAGS += -I$(XI_CC3220_PATH_SDK)/source/ti/devices/cc32xx/driverlib
 XI_COMPILER_FLAGS += -I$(XI_CC3220_PATH_SDK)/source
 XI_COMPILER_FLAGS += -I$(XI_CC3220_PATH_SDK)/source/ti/drivers
 XI_COMPILER_FLAGS += -I$(XI_CC3220_PATH_SDK)/source/ti/drivers/net/wifi
+XI_COMPILER_FLAGS += -I$(XI_CC3220_PATH_SDK)/source/ti/drivers/net/wifi/sys
```

Comment:
I know you haven't touched it, but please remove the line 8 and 9:

```
# todo: this is common, move it to outside
XI ?= $(XI_BINDIR)/libxively.a
```

It isn't needed anymore. Since the copy-initialize this file from mt-cc3200.mk this issue has been solved.
Thanks.